draft-iyengar-quic-delayed-ack.md

```diff
@@ -310,8 +307,9 @@ disable_ack_on_reordering (0xXXXX):
 
 As specified in Section 13.3.1 of {{QUIC-TRANSPORT}}, an endpoint SHOULD
 immediately acknowledge packets marked with the ECN Congestion Experienced (CE)
-codepoint in the IP header. Doing so reduces the peer's response time to
-congestion events.
+codepoint in the IP header, even if the `disable_ack_on_reordering` transport
+parameter is present. Doing so reduces the peer's response time to congestion
```

Comment:
I don't think this is necessary. The TP says nothing about ECN, and ECN says nothing about reordering.

Reply:
Removed.